scanpy/tests/notebooks/test_pbmc3k.py

```diff
@@ -29,6 +29,7 @@
 @needs.leidenalg
 def test_pbmc3k(image_comparer):
     save_and_compare_images = partial(image_comparer, ROOT, tol=20)
+    save_and_compare_images_rank_genes = partial(image_comparer, ROOT, tol=10) # 20 is too high for such sparse plots
```

Comment:
Do we think this could be a bug elsewhere, tolerance too low for picking up errors?  I can certainly open an issue for it.